vars/buildDaosJob.groovy

```diff
@@ -0,0 +1,32 @@
+// vars/buildDaosJob.groovy
+
+  /**
+   * buildDaosJob step method.
+   *
+   * This is intended only for use in a Jenkinsfile in a Matrix stage.
+   * Builds a branch of DAOS based on env.TEST_BRANCH.
+   *
+   * The paramater names must match parameters that are the Jenkinsfile for
+   * DAOS master.
+   *
+   * @param config Map of parameters passed
+   *
+   */
+
+Map call(Map config = [:]) {
+    build job: 'daos-stack/daos/' + setupDownstreamTesting.test_branch(env.TEST_BRANCH),
```

Comment:
```suggestion
Map call(String branch, Integer priority) {
    build job: 'daos-stack/daos/' + setupDownstreamTesting.test_branch(branch),
```

But does `build()` actually return a `Map`?  If not, this can probably just be `void`:

```suggestion
void call(String branch, Integer priority) {
    build job: 'daos-stack/daos/' + setupDownstreamTesting.test_branch(branch),
```

Reply:
Changed in current push